app.py

```diff
@@ -0,0 +1,103 @@
+from os import read, write
+from flask import Flask, request, jsonify, render_template
+import flask
+import json
+from jsonschema import validate
+import jsonschema
+from flask_ngrok import run_with_ngrok
+
+
+
+app = Flask(__name__)
+app.config["DEBUG"] = True
+# run_with_ngrok(app)
+
+#Loading schema to a variable
+def load_schema():
+    schema = json.load(open('schema.json'))
+    return schema
+#This function goes trough the keys(id's) inside users file and get's the last id, then parses it to int, adds a counter and returns the new ID as a string
+def get_index():
+    index = int((list(load_users().keys()))[-1]) + 1
+    return str(index)
+    
+#Reading users data from json file and storing them as variable
+def load_users():
+    json_object = json.load(open('users.json'))
```

Comment:
When opening a file you need to use context manager, which will close the file for you after you exit its context. Otherwise (like here) you're left with an opened file that never got closed.